lib/ResponseBuilder.js

```diff
@@ -1,8 +1,7 @@
 'use strict';
 
 const AWS = require('aws-sdk');
-const BPromise = require('bluebird');
-const request = require('request-promise');
+const got = require('got');
```

Comment:
way smaller, request includes like ~600kb of cert detection and other weird things